client/about/index.js

```diff
@@ -0,0 +1,38 @@
+import { render, html } from 'lit'
+import './style.scss'
+
+//https://magenta.tensorflow.org/studio/
+
+function handleOpenWebsite(e){
+	e.preventDefault()
+	fetch(e.target.href)
```

Comment:
does this line do anything with the fetched website?

Reply:
yes! so in CEF, there's no way to open a link in the default browser, it just opens in the CEF window. what this does is makes a request to the express server, which then Max opens the link in the default browser. i'll add a comment here because yeah, it seems like this does nothing.